process/process.js

```diff
@@ -1,17 +1,22 @@
 const GameData = require('./src/GameData');
 
-const gameData = new GameData();
+const gameData = new GameData(__dirname);
 
 if (process.argv.includes('download')) {
   console.log("Downloading data...");
-  gameData.download();
+  gameData.download("https://github.com/jasonrohrer/OneLifeData7.git");
```

Comment:
If we can pull this out to a const up the top, will be a little nicer for UncleGus to keep his fork in sync.

Reply:
How about using an ENV variable which falls back to this? Then no code change will be necessary. I'll try this.